core/class_db.cpp

```diff
@@ -416,8 +416,8 @@ uint64_t ClassDB::get_api_hash(APIType p_api) {
 			snames.sort_custom<StringName::AlphCompare>();
 
 			for (List<StringName>::Element *F = snames.front(); F; F = F->next()) {
-				hash = hash_djb2_one_64(F->get().hash(), hash);
-				hash = hash_djb2_one_64(t->constant_map[F->get()], hash);
+				hash = hash_djb2_one_64(uint64_t(F->get().hash()), hash);
+				hash = hash_djb2_one_64(uint64_t(t->constant_map[F->get()]), hash);
```

Comment:
Does it still produce the same API hash?

Reply:
An identical result should be displayed, since the secret conversion to uint64_t has been converted to explicit, without any change in logic.